### ISSUE TITLE
Tasks built based on coding challenges 

### ISSUE THREAD

**Issue by danyaljj** [allenai/natural-instructions#127]
We can use the task description of coding challenges (the ones that are relatively simple to understand for avg humans) and create input/output instances based on the solution program. 

https://leetcode.com/
https://www.codechef.com/problems/school
https://edabit.com/challenges

**Comment by swarooprm** [comment]
FYI: @DZuoShi

**Comment by swarooprm** [comment]
FYI: @maykillmore

**Comment by eshaanpathak** [comment]
This may also be helpful or relevant, though it is pseudocode to code and could also go vice versa: https://sumith1896.github.io/spoc/

Time to make our own OpenAI Codex lol.

**Comment by ayush-anand-13** [comment]
Yes I'll start working on this. Just submitted a big assignment for 515 and
I can get on this.

**Comment by swarooprm** [comment]
I guess @Ravsehajsinghpuri is also working on it.

**Comment by swarooprm** [comment]
Make sure to create a PR soon so that your tasks get registered. 
Note that, if multiple people create the exact same task, we usually accept the PR created first and ask the other person to change their PR.
